### PR TITLE
Removed some logs for a few percent faster inference

### PR DESCRIPTION
**Pull request by SkBlaz**

### CODE REVIEW

File: src/model_instance.rs [hunk 1]
```diff
@@ -447,9 +447,9 @@ impl ModelInstance {
 			}
 		}
 
-		for (hyper_name, hyper_value) in replacement_hyperparam_ids.into_iter() {
-			println!("Warning! Updated hyperparameter {} to value {}", hyper_name, hyper_value);
-		}
+		// for (hyper_name, hyper_value) in replacement_hyperparam_ids.into_iter() {
```
Comment (adischw) [inline src/model_instance.rs:450]: i think its a bad idea to remove this log. we have to have visibility if we update the HP

Comment (SkBlaz) [reply src/model_instance.rs]: Only commented it for this version; however, this does not seem to be the part responsible for most prints indeed, re-added.

File: src/model_instance.rs [hunk 2]
```diff
@@ -407,7 +407,7 @@ impl ModelInstance {
 		/*! A method that enables updating hyperparameters of an existing (pre-loaded) model.
 		Currently limited to the most commonly used hyperparameters: ffm_learning_rate, ffm_power_t, power_t, learning_rate. */
 		
-		println!("Replacing initial regressor's hyperparameters from the command line ..");
+//		println!("Replacing initial regressor's hyperparameters from the command line ..");
```
Comment (adischw) [inline src/model_instance.rs:410]: this is also important to add and have visibility of

Comment (SkBlaz) [reply src/model_instance.rs:410]: I actually think this one is redundant. It will print either way, even if there are no replacements, it seems

Comment (adischw) [reply src/model_instance.rs:410]: ok so lets add a print to those if cases because we need to understand which values were changed

**Comment by bbenshalom** [comment]
I suggest deleting rather than leaving as comments. Either we need these logs or we don't.
Another (far better IMO) option is to use a logger (see [this ](https://docs.rs/log/latest/log/) crate) and instead of removing the prints, make them debug loggings.

**Comment by ggaspersic** [comment]
Agree with @bbenshalom, if not needed remove them & prefer using a logging framework. Can also be configured to expose data in a format that will be more easily ingestible by logstash & exposed via Kibana. & if performance is a very big issue then an async logging framework can be used in place.

**Comment by SkBlaz** [comment]
Agreed, let's do it this way

**Pull request closed without merging**